src/App.js

```diff
@@ -34,11 +37,20 @@ function App() {
       {isLoading ? (
         <p>Loading...</p>
       ) : (
-        <ConferenceList conferences={conferences} />
+      <Router>
```

Comment:
I'm not sure but I believe it might be better to move the Loading logic into the router so it would look something like that:

`<Router>
   {isLoading ? ( <p>Loading....</p>):(<Switch>......[Rest of switch markup]</Switch></Router>`